### PR TITLE
Changed LICENSE to MIT

### PR DESCRIPTION
**Pull request by mosheBelostotsky**
Added LICENSE template generation (will prompts)

### CODE REVIEW

File: generators/script/index.js [hunk 1]
```diff
@@ -125,17 +139,18 @@ var HubotScriptGenerator = yeoman.generators.Base.extend({
 
       this.mkdir('src');
       this.template('src/template.coffee', 'src/' + this.scriptName + '.coffee');
-	  
-	  this.mkdir('lib');
-	  
+
+      this.mkdir('lib');
```
Comment (rqc) [inline generators/script/index.js]: @mosheBelostotsky could you elaborate why do we need a `lib` folder now? This doesn't seem to be the default directory structure of most `hubot` scripts out there.

**Comment by mosheBelostotsky** [comment]
@rqc we decided to add lib dir for non bot code.
in this dir one can place code that not extends robot.

File: generators/script/index.js (hunk 1 above)
Comment (mosheBelostotsky) [inline generators/script/index.js]: @rqc we decided to add lib dir for non bot code.
in this dir one can place code that not extends robot.

Comment (rqc) [inline generators/script/index.js]: I think this adds more boilerplate which might not be used. If a developer wants to add nested folders within `src` that should by fine. Typically this is how `js` / `nodejs` projects are structured. 

Please remove.

**Comment by mosheBelostotsky** [comment]
@rqc I merged with HEAD and removed lib.

**Comment by rqc** [comment]
@mosheBelostotsky looks good.

**Pull request merged**